tests/layer_tests/pytorch_tests/test_bitwise_ops.py

```diff
@@ -140,3 +140,61 @@ def test_bitwise_operators(self, lhs_dtype, rhs_dtype, lhs_shape, rhs_shape, ie_
             trace_model=True,
             freeze_model=False,
         )
+
+
+class TestBitwiseInplaceOp(PytorchLayerTest):
+    def _prepare_input(self, lhs_shape, rhs_shape, dtype):
+        choices = np.array([0, 1, 255, 7])
+        x = np.random.choice(choices, lhs_shape).astype(dtype)
+        y = np.random.choice(choices, rhs_shape).astype(dtype)
+        return x, y
+
+    def create_model(self, op):
+        class aten_bitwise(torch.nn.Module):
+            def __init__(self, op) -> None:
+                super().__init__()
+                if op == "aten::__ior__":
+                    self.forward = self.forward_or
+                if op == "aten::__iand__":
+                    self.forward = self.forward_and
+                if op == "aten::__ixor__":
+                    self.forward = self.forward_xor
+
+            def forward_or(self, lhs, rhs):
+                return lhs.__ior__(rhs)
+    
+            def forward_and(self, lhs, rhs):
+                return lhs.__iand__(rhs)
+    
+            def forward_xor(self, lhs, rhs):
+                return lhs.__ixor__(rhs)
+
+        return aten_bitwise(op), None, op
+
+    @pytest.mark.nightly
+    @pytest.mark.precommit
+    @pytest.mark.parametrize("dtype", ["bool", "int32"])
```

Comment:
can we run these tests for export and fx?

Reply:
it does not support inplace ops as i know

Reply:
right, not needed then